### PR TITLE
added initial create shopping list page

### PR DESCRIPTION
**Pull request by mongy910**
## Status: 

:construction: In development

## Description
The initial version of the create shopping list page

Related issues: #56

### CODE REVIEW

File: frontend/pages/SearchPage.js [hunk 1]
```diff
@@ -20,10 +35,84 @@ export default class SearchPage extends React.Component {
...
+      itemList: newList
+    })
+    this.selectCheapestStore()
+  }
+
+  // choose cheapest store out of the items
+  selectCheapestStore = () => {
+    // uses store name as key
+    storeTotals = {}
+    for (var i = 0; i < this.state.itemList.length; i++) {
```
Comment (davidenwang) [inline frontend/pages/SearchPage.js]: change var to let

File: frontend/pages/SearchPage.js [hunk 2]
```diff
@@ -20,10 +35,84 @@ export default class SearchPage extends React.Component {
...
+    return -1
+  }
+
+  
+
   render() {
     return (
       <View style={styles.container}>
-        <ItemSearchBar onSearch={this.submitSearch}/>
+        <ItemSearchBar onSearch={this.submitSearch} />
```
Comment (davidenwang) [inline frontend/pages/SearchPage.js:115]: looks good 📜

File: frontend/pages/SearchPage.js [hunk 3]
```diff
@@ -33,6 +122,42 @@ export default class SearchPage extends React.Component {
...
+        <View style={styles.footer}>
+          <ListItem
+            title={'Total (' + this.state.itemList.length + ' items)'}
+            subtitle={this.state.selectedStore}
+            rightTitle={'$' + this.state.currentTotal}
+            topDivider
+          />
+        </View>
+        <View>
+          <Button
```
Comment (davidenwang) [inline frontend/pages/SearchPage.js:149]: good job with the ✨ button ✨ i like the icon

**Review by davidenwang — CHANGES_REQUESTED** [review]
Not bad you're nasty 🤢keep up the good work and just do a quick var to let change

File: frontend/pages/SearchPage.js (hunk 1 above)
Comment (mongy910) [reply frontend/pages/SearchPage.js]: got it

**Review by davidenwang — APPROVED** [review]

**Pull request merged**